livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -549,5 +554,5 @@ def _to_deepgram_url(opts: dict, *, websocket: bool = False) -> str:
 
     # lowercase bools
     opts = {k: str(v).lower() if isinstance(v, bool) else v for k, v in opts.items()}
-    base_url = BASE_URL_WS if websocket else BASE_URL
+    base_url = self._base_url_ws if websocket else self._base_url
```

Comment:
`self` doesn't work when outside of a class

Reply:
Sorry that was a silly overlook - updated